trulens_eval/trulens_eval/feedback/provider/dummy.py

```diff
@@ -0,0 +1,56 @@
+import json
```

Comment:
why is this here instead of in the stuff with all the other dummy things.

Reply:
dummy_app is just an app. There is also dummy provider and dummy endpoint.

Reply:
These are more likely to be used by users if they want to test their apps with feedback functionality without actually incurring any endpoint costs (using a dummy provider).